[Tech debt] Fix logs in tests

There are way too many lines of logs in tests which make it difficult to see which tests have failed or succeeded

Also they output wrong error messages at the end of the run (probably docker containers stopped before the tested application has stopped?)
```
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to localhost:9324 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused (Connection refused)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
```

Last problem is that the startup tests also output a bunch of errors. These are due to the fact that the StartupTests start the applications with the real client modules that point to our real AWS, but they don't have (and shouldn't have) the extra configuration needed to access the resources (ie. arns of streams, queue names, table names, etc). Example from id_minter:
```
om.amazonaws.services.sqs.model.QueueDoesNotExistException: The specified queue does not exist for this wsdl version. (Service: AmazonSQS; Status Code: 400; Error Code: AWS.SimpleQueueService.NonExistentQueue; Request ID: c5442172-b08a-508d-84a5-700ea0b020d5)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1588)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1258)
```

--- a/calm_adapter/src/test/scala/uk/ac/wellcome/calm_adapter/StartupTest.scala
+++ b/calm_adapter/src/test/scala/uk/ac/wellcome/calm_adapter/StartupTest.scala
@@ -2,14 +2,15 @@ package uk.ac.wellcome.platform.calm_adapter
 
 import com.google.inject.Stage
 import com.twitter.finatra.http.EmbeddedHttpServer
-import com.twitter.inject.server.WordSpecFeatureTest
+import com.twitter.inject.server.FeatureTest
+import uk.ac.wellcome.test.utils.StartupLogbackOverride
 
-class StartupTest extends WordSpecFeatureTest {
+class StartupTest extends FeatureTest with StartupLogbackOverride  {
 
   val server = new EmbeddedHttpServer(stage = Stage.PRODUCTION,
                                       twitterServer = new Server)
 
-  "server" in {
+  test("server starts up correctly") {
     server.assertHealthy()
   }
 }

--- a/common/src/test/resources/logback-startup-test.xml
+++ b/common/src/test/resources/logback-startup-test.xml
@@ -1,0 +1,13 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <!-- encoders are assigned the type
+             ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="OFF">
+        <appender-ref ref="STDOUT" />
+    </root>
+</configuration>

--- a/common/src/test/scala/uk/ac/wellcome/test/utils/StartupLogbackOverride.scala
+++ b/common/src/test/scala/uk/ac/wellcome/test/utils/StartupLogbackOverride.scala
@@ -1,0 +1,19 @@
+package uk.ac.wellcome.test.utils
+
+import ch.qos.logback.classic.LoggerContext
+import ch.qos.logback.classic.joran.JoranConfigurator
+import org.apache.commons.io.IOUtils
+import org.slf4j.LoggerFactory
+
+import scala.tools.nsc.classpath.FileUtils
+
+trait StartupLogbackOverride {
+  val loggerContext =
+    LoggerFactory.getILoggerFactory.asInstanceOf[LoggerContext]
+  loggerContext.reset
+  val configurator = new JoranConfigurator
+  configurator.setContext(loggerContext)
+  configurator.doConfigure(
+    getClass
+      .getResourceAsStream("/logback-startup-test.xml"))
+}

--- a/common/src/test/scala/uk/ac/wellcome/test/utils/StartupLogbackOverride.scala
+++ b/common/src/test/scala/uk/ac/wellcome/test/utils/StartupLogbackOverride.scala
@@ -2,10 +2,7 @@ package uk.ac.wellcome.test.utils
 
 import ch.qos.logback.classic.LoggerContext
 import ch.qos.logback.classic.joran.JoranConfigurator
-import org.apache.commons.io.IOUtils
 import org.slf4j.LoggerFactory
-
-import scala.tools.nsc.classpath.FileUtils
 
 trait StartupLogbackOverride {
   val loggerContext =

--- a/id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/StartupTest.scala
+++ b/id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/StartupTest.scala
@@ -4,8 +4,9 @@ import com.google.inject.Stage
 import com.twitter.finatra.http.EmbeddedHttpServer
 import com.twitter.inject.server.FeatureTest
 import com.twitter.finagle.http.Status._
+import uk.ac.wellcome.test.utils.StartupLogbackOverride
 
-class StartupTest extends FeatureTest {
+class StartupTest extends FeatureTest with StartupLogbackOverride {
 
   val server = new EmbeddedHttpServer(stage = Stage.PRODUCTION,
                                       twitterServer = new Server)

--- a/ingestor/src/test/scala/uk/ac/wellcome/platform/ingestor/StartupTest.scala
+++ b/ingestor/src/test/scala/uk/ac/wellcome/platform/ingestor/StartupTest.scala
@@ -3,8 +3,9 @@ package uk.ac.wellcome.platform.ingestor
 import com.google.inject.Stage
 import com.twitter.finatra.http.EmbeddedHttpServer
 import com.twitter.inject.server.FeatureTest
+import uk.ac.wellcome.test.utils.StartupLogbackOverride
 
-class StartupTest extends FeatureTest {
+class StartupTest extends FeatureTest with StartupLogbackOverride {
 
   val server = new EmbeddedHttpServer(stage = Stage.PRODUCTION,
                                       twitterServer = new Server)

--- a/transformer/src/test/scala/uk/ac/wellcome/transformer/StartupTest.scala
+++ b/transformer/src/test/scala/uk/ac/wellcome/transformer/StartupTest.scala
@@ -3,8 +3,9 @@ package uk.ac.wellcome.platform.transformer
 import com.google.inject.Stage
 import com.twitter.finatra.http.EmbeddedHttpServer
 import com.twitter.inject.server.FeatureTest
+import uk.ac.wellcome.test.utils.StartupLogbackOverride
 
-class StartupTest extends FeatureTest {
+class StartupTest extends FeatureTest with StartupLogbackOverride {
 
   val server = new EmbeddedHttpServer(
     stage = Stage.PRODUCTION,

